### PR TITLE
[AL-7548] Improve test coverage for send to annotate

### PR DESCRIPTION
**Pull request by richardsun0713**

### CODE REVIEW

File: tests/integration/annotation_import/conftest.py [hunk 1]
```diff
@@ -579,6 +579,21 @@ def configured_project(client, initial_dataset, ontology, rand_gen, image_url):
...
+def project_with_ontology(client, configured_project, ontology, rand_gen):
+    project = client.create_project(name=rand_gen(str),
+                                    queue_mode=QueueMode.Batch,
+                                    media_type=MediaType.Image)
+    editor = list(
+        client.get_labeling_frontends(
+            where=LabelingFrontend.name == "editor"))[0]
+    project.setup(editor, ontology)
+
+    yield [project, ontology]
```
Comment (vbrodsky) [inline tests/integration/annotation_import/conftest.py:592]:
```suggestion
    yield project, ontology # do not think need to wrap it in an array, i.e leave it as a tuple
```

File: tests/integration/annotation_import/test_send_to_annotate_mea.py [hunk 2]
```diff
@@ -13,15 +14,38 @@ def test_send_to_annotate_from_model(client, configured_project,
...
     task = model_run.send_to_annotate_from_model(
         destination_project_id=destination_project.uid,
         batch_name="batch",
         data_rows=UniqueIds(data_row_ids),
-        task_queue_id=initial_labeling_task.uid,
-        params={})
+        task_queue_id=initial_review_task.uid,
+        params={
+            "predictions_ontology_mapping":
+                ontology_mapping,
```
Comment (vbrodsky) [inline tests/integration/annotation_import/test_send_to_annotate_mea.py:45]: so this is basically {id1: id1, id2: id2... } right? Kind of weird mapping to itself. Should be test a case when we have like {id1: id2, id3: id4} ?

File: tests/integration/annotation_import/test_send_to_annotate_mea.py [hunk 3]
```diff
@@ -32,3 +56,7 @@ def test_send_to_annotate_from_model(client, configured_project,
     destination_data_rows = list(destination_batches[0].export_data_rows())
     assert len(destination_data_rows) == len(data_row_ids)
     assert all([dr.uid in data_row_ids for dr in destination_data_rows])
+
+    # Since data rows were added to a review queue, predictions should be imported into the project as labels
+    destination_project_labels = (list(destination_project.labels()))
+    assert len(destination_project_labels) == len(data_row_ids)
```
Comment (vbrodsky) [inline tests/integration/annotation_import/test_send_to_annotate_mea.py:62]: Possible also to compare some label attributes? Also to make sure len(...) is not 0?

File: tests/integration/annotation_import/test_send_to_annotate_mea.py (hunk 2 above)
Comment (richardsun0713) [reply tests/integration/annotation_import/test_send_to_annotate_mea.py:45]: This is one of the valid use-cases. I can add additional tests as a follow-up

**Review by vbrodsky — APPROVED** [review]

**Pull request merged**